src/types.rs

```diff
@@ -287,18 +293,23 @@ impl Into<&str> for &MediaType {
             MediaType::H265 => MEDIA_TYPE_H265,
             MediaType::AAC => MEDIA_TYPE_AAC,
             MediaType::TTXT => MEDIA_TYPE_TTXT,
+            MediaType::VP9 => MEDIA_TYPE_VP9,
```

Comment:
Can this match the ordering from above? Right after H265 type.

Reply:
Sure.